src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
I definitely need an UI Link component due to the Javascript binding/signaling features as written above. And in contrast to the Divider I think a renderer for a "stand alone" link would also be feasible. We might limit the use of where links can appear drastically by rules for now.